src/pages/experiments/[experimentId]/data-processing/data-integration/components/plots/ElbowPlot.jsx

```diff
@@ -0,0 +1,133 @@
+import React from 'react';
+import { Vega } from 'react-vega';
+import PropTypes from 'prop-types';
+
+const ElbowPlot = (props) => {
```

Comment:
shouldn't this be stored under ` src/components/plots`, next to the Frequency plot?

Reply:
I didn't store it there because it wasn't shared anywhere else in the app, but if you believe it makes more sense to have it there I can do that